assets/prometheus-operator-user-workload/service-monitor.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app.kubernetes.io/name: prometheus-operator
     app.kubernetes.io/version: v0.33.0
   name: prometheus-operator
-  namespace: openshift-user-workload-monitoring
+  namespace: openshift-monitoring
```

Comment:
hmm :D why was this updated automatically incorrectly for me? 🤔

Reply:
hmm, can you make sure that the changes from https://github.com/coreos/kube-prometheus/pull/260 is pulled in your local jsonnet vendor directory?

Reply:
And it seems to be taking the latest commit of kube-prometheus below.